### ISSUE TITLE
Can not upload bigger images;

### ISSUE THREAD

**Issue by mat1th** [parley-messaging/ios-library#56]
**Describe the bug**
I'm trying the demo app that is included in the project. I've found a issue with uploading images. 

**To Reproduce**
1. Start the example app.
2. Open a new chat.
3. Upload a small image. (for example a image from ~5 KB)
4. See that uploading the image does work. (Checkmark shown that image is send)
5. Upload 1 of the default images of the iOS simulator.
6. See that upload is failed. (Cross icon shown on image that you tried to upload).

**Expected behaviour**
I do expect that the SDK allows uploading all kinds of sizes of images, and if the image is to bit it will give me an error before uploading, or at least give me the reason why my image could not been uploaded.

**Screenshots**
![screenshot_image_failed](https://github.com/parley-messaging/ios-library/assets/6486389/6f222665-bca4-4689-94f7-e28a764ab9c3)

**Additional context**
The backed does give me the following response on the image I did try uploading: 
```json
{
  "notifications": [
    {
      "type": "error",
      "message": "invalid_media_type"
    }
  ],
  "status": "ERROR",
  "metadata": {
    "values": {
      "url": "media"
    },
    "method": "post",
    "duration": 0.011
  }
}
```
and the status code is: 400.

**Comment by alexkok** [comment]
Thanks for reporting and the clear description @mat1th ! 
We will look into it and check how we'll resolve this.

**Comment by alexkok** [comment]
Released in 3.9.1 🚀
